default_filename.json

```diff
@@ -0,0 +1 @@
+{"persons":[{"name":{"fullName":"Bernice Yu"},"phone":{"value":"99272758"},"email":{"value":"berniceyu@example.com"},"address":{"value":"Blk 30 Lorong 3 Serangoon Gardens, #07-18"},"tags":[{"tagName":"friend"}]},{"name":{"fullName":"David Li"},"phone":{"value":"91031282"},"email":{"value":"lidavid@example.com"},"address":{"value":"Blk 436 Serangoon Gardens Street 26, #16-43"},"tags":[{"tagName":"friend"}]}]}
```

Comment:
It would be better if this can be formatted, instead of all the information being in one line.